### PR TITLE
Extra set of units

### PR DESCRIPTION
**Pull request by evertoncolling**
This PR adds some extra units for the following quantities:
- Length
- Velocity
- Angle
- Energy Per Area
- Power Per Area

### CODE REVIEW

File: versions/v1/units.json [hunk 1]
```diff
@@ -1642,5 +1642,842 @@
...
+        "conversion": {
+            "multiplier": 11356.5267,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/BTU_IT-PER-SEC-FT2"
+    },
+    {
+        "externalId": "power_per_area:w-per-ft2",
+        "name": "W/ft\u00b2",
```
Comment (polomani) [inline versions/v1/units.json:2469]:
```suggestion
        "name": "W/ft²",
```
for the characters that are allowed in JSON, could we just use the non-escaped form?

Comment (polomani) [reply versions/v1/units.json:2469]: we should be able to use `json.dumps(json.loads(test), ensure_ascii=False)` in python to achieve this
(if you are using python for this?)

Comment (evertoncolling) [reply versions/v1/units.json:2469]: Sure, not all characters support this, but we can try to adhere to this pattern when possible

Comment (evertoncolling) [reply versions/v1/units.json]: Done, your suggestion worked like a charm!

File: versions/v1/unitSystems.json [hunk 2]
```diff
@@ -104,6 +140,22 @@
...
+                "name": "Length",
+                "unitExternalId": "length:ft"
+            },
+            {
+                "name": "Velocity",
+                "unitExternalId": "velocity:ft-per-sec"
+            },
+            {
+                "name": "Energy Per Area",
+                "unitExternalId": "energy_per_area:ft-lb_f-per-ft2"
```
Comment (polomani) [inline versions/v1/unitSystems.json:154]: I might have missed some of the discussions here, but are we using a mix of kebab and snake cases? in what cases?

Comment (evertoncolling) [reply versions/v1/unitSystems.json:154]: The `unitExternalId` is formed as `{quantity}:{qudtName}`, where we apply lowercase and replace whitespaces from both `quantity` and `qudtName` with a underscore (`_`). There are cases where the `qudtName` contains a hyphen (`-`) which is kept today.

**Review by polomani — APPROVED** [review]
LGTM

**Pull request merged**